packages/apps/human-app/server/src/integrations/reputation-oracle.service.ts

```diff
@@ -0,0 +1,40 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { HttpService } from '@nestjs/axios';
+import { lastValueFrom } from 'rxjs';
+import { SignupWorkerDto } from '../interfaces/signup-worker-request.dto';
+
+@Injectable()
+export class ReputationOracleService {
```

Comment:
By naming convention agreement (see README) - this should be Gateway

Reply:
Fixed in commit db098042